include/language.php

```diff
@@ -405,11 +405,11 @@ public static function get_flag_information( $code ) {
 		 * }
 		 * @param string $code Flag code.
 		 */
-		$flag = apply_filters( 'pll_flag', $flag, $code );
+		$flag = apply_filters( 'pll_flag', $default_flag, $code );
 
 		$flag['url'] = esc_url_raw( $flag['url'] );
 
-		if ( empty( $flag['src'] ) ) {
+		if ( empty( $flag['src'] ) || ( $flag['src'] === $default_flag['src'] && $flag['url'] !== $default_flag['url'] ) ) {
```

Comment:
`$default_flag['src']` may not exist if `PLL_ENCODED_FLAGS` is not defined. Maybe it should be set L376.

Reply:
Good catch. I didn't pay attention that PHPStan warned about that due to other errors.